jetty9/src/main/docker/Dockerfile

```diff
@@ -39,8 +39,9 @@ RUN mkdir webapps $TMPDIR \
  && chown -R jetty:jetty $JETTY_BASE $TMPDIR
 
 # Start Jetty directly
-COPY app-env.bash /
-RUN chmod +rx /app-env.bash
+COPY setup-env-ext.bash /tmp/
+RUN cat /tmp/setup-env-ext.bash >> /setup-env.bash \
+ && rm /tmp/setup-env-ext.bash
```

Comment:
I still think it is worth removing it just to avoid duplication, which may confuse people looking at the image and trying to extend/debug it.   I can imagine somebody editing that file in a DockerFile and then being surprised it is not actually used.